plugins/module_utils/nae.py

```diff
@@ -1293,6 +1305,27 @@ def get_compliance_object(self, name):
             self.module.exit_json(
                 msg="WARNING: {0} {1} does not exist!!!".format(type_map.get(self.params.get('selector')), self.params.get('name')), **self.result)
 
+    def query_compliance_object(self, name):
```

Comment:
This function should also be used in get_compliance_object() because you know have duplicate code.